analyses/fusion-summary/01-fusion-summary.Rmd

```diff
@@ -44,6 +44,30 @@ filterFusion <- function(df, bioid, fuses, genes) {
   return(df %>% select(Sample, FusionName))
 }
 
+#create function specifically for lgat, since we want more information from the putative oncogenic file
+filterLGATFusion <- function(df, bioid, fuses, genes) {
+  if (!missing(bioid)) {
+    df <- filter(df, Sample %in% bioid)
+  }
+  if (!missing(fuses) & !missing(genes)) {
+    df <- filter(df, FusionName %in% fuses |
+                   Gene1A %in% genes |
+                   Gene2A %in% genes |
+                   Gene1B %in% genes |
+                   Gene2B %in% genes)
+  } else if (!missing(fuses)) {
+    df <- filter(df, FusionName %in% fuses)
+  } else if (!missing(genes)) {
+    df <- filter(df,
+                 Gene1A %in% genes |
+                   Gene2A %in% genes |
+                   Gene1B %in% genes |
+                   Gene2B %in% genes)
+  }
+  return(df %>% 
+           select(Sample, FusionName, Fusion_Type, Gene1A, Gene1B, Gene2A,  Gene2B, Gene1A_anno, Gene1B_anno, LeftBreakpoint, RightBreakpoint,      reciprocal_exists, DomainRetainedGene1A, DomainRetainedGene1B) %>%
```

Comment:
For this, since we are specifically looking for in-frame (`Fusion_Type`), kinase (`Gene1A_anno`, `Gene1B_anno`), whether a reciprocal exists (`reciprocal_exists `), and fusions with kinase domains retained (`DomainRetainedGene1A`, `DomainRetainedGene1B`), these columns required for LGAT fusion compilation. I added the left and right breakpoint columns to make sure that I was seeing unique fusions, although the rows may be unique enough with the combination of columns above. Let me check on this.

Reply:
Looks like the number of unique fusions without breakpoints == 265 and with breakpoints == 300, but the results are identical. I will remove those two columns!